dask/bytes/core.py

```diff
@@ -1,24 +1,14 @@
 import os
 import copy
 
-from fsspec.core import (  # noqa: F401
-    OpenFile,  # noqa: F401
-    open_files,  # noqa: F401
-    get_fs_token_paths,  # noqa: F401
-    expand_paths_if_needed,  # noqa: F401
-    _expand_paths,  # noqa: F401
-    get_compression,  # noqa: F401
+from fsspec.core import (
+    OpenFile,
+    get_fs_token_paths,
```

Comment:
Nitpick: I wonder if `black` will let us put this on a single line

Reply:
I bet it would :)